okteto.yml

```diff
@@ -0,0 +1,18 @@
+name: hello-world
+build:
+  hello-world:
+    context: .
+    dockerfile: Dockerfile
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    image: okteto/dev:latest
+    autocreate: true
+    command: npm i && npm run start
+    workdir: /usr/src/app
```

Comment:
this shouldn't be needed

Reply:
those are removed, command I left with bash